.github/workflows/hardhat-ci.yml

```
@@ -71,16 +71,16 @@ jobs:
       matrix:
         package: ${{ fromJson(needs.list-packages.outputs.packages) }}
         os: [ubuntu-latest, macos-latest, windows-latest]
-        node: [20, 22, 24]
+        node: [20.0.0, 20, 22, 24]
```

I am unsure whether we should go for the minimum version of each major node version as well?

```suggestion
        node: [20.0.0, 20, 22.0.0, 22, 24.0.0, 24]
```